FireRender.Maya.Src/FireRenderUtils.cpp

```diff
@@ -2171,3 +2173,71 @@ void EnableAOVsFromRSIfEnvVarSet(FireRenderContext& context, FireRenderAOVs& aov
 
 	aovs.applyToContext(context);
 }
+
+#ifdef __APPLE__ // https://stackoverflow.com/questions/31346887/header-for-environ-on-mac
+extern char **environ;
+#endif
+
+class EnvironmentVarsWrapper
+{
+public:
+	static const std::map<std::string, std::string>& GetEnvVarsTable(void)
+	{
+		static EnvironmentVarsWrapper instance;
+
+		return instance.m_eVars;
+	}
+
+	EnvironmentVarsWrapper(EnvironmentVarsWrapper const&) = delete;
+	void operator=(EnvironmentVarsWrapper const&) = delete;
+
+private:
+	EnvironmentVarsWrapper(void) 
+	{
+		char *s = *environ;
+		int i = 1;
+		for (; s; i++)
+		{
+			std::string tmp(s);
+			std::string varName = tmp.substr(0, tmp.find("="));
```

Comment:
please keep index we get from find function to avoid double call of this find function